api/libproject/devstart.rb

```diff
@@ -2708,3 +2708,40 @@ def randomize_vcf(cmd_name, *args)
     "random copies in a given output directory",
   :fn => ->(*args) { randomize_vcf("randomize-vcf", *args) }
 })
+
+def set_access_module_timestamps(cmd_name, *args)
```

Comment:
should this be named `expire_` instead? Or do you intend to make this more generic later?

Reply:
right - I expect that this will involve into something that does more